packages/lesswrong/components/ea-forum/digest/EditDigest.tsx

```diff
@@ -172,7 +172,7 @@ const EditDigest = ({classes}:{classes: ClassesType}) => {
   const digest = results?.[0]
 
   // get the list of posts eligible for this digest
-  const { data } = useQuery(gql`
+  const { data, refetch } = useQuery(gql`
```

Comment:
This does in fact take an input, and will get called with a different input than what it's expecting

Reply:
Turns out we don't need this anyway so I removed it